resources.go

```diff
@@ -373,6 +404,7 @@ func Provider() tfbridge.ProviderInfo {
 			"google_compute_instance_group_manager":        {Tok: gcpResource(gcpCompute, "InstanceGroupManager")},
 			"google_compute_instance_template":             {Tok: gcpResource(gcpCompute, "InstanceTemplate")},
 			"google_compute_interconnect_attachment":       {Tok: gcpResource(gcpCompute, "InterconnectAttachment")},
+			"google_compute_managed_ssl_certificate":       {Tok: gcpResource(gcpCompute, "MangedSslCertificate")},
```

Comment:
We use uppercase SSL elsewhere, like `SSLCertificate` and `SSLPolicy` - how about `ManagedSSLCertificate`?

Reply:
I think unfortunately we've been really inconsistent on this.  In the AWS provider, my sense is we've nearly always used `Ssl` for three letter acronyms.  Would love to do an audit on this for our next round of breaking changes.

Reply:
After discussion with @lukehoban it sounds like we'll go for these next time around after we decide a project-wide policy.